.semaphore/npm.yml

```diff
@@ -21,4 +21,5 @@ blocks:
         - name: yarn publish
           commands:
             - echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-            - yarn publish --access=public
+            # TODO: Remove the `--tag=next` specifier when we're ready to release v3.0.0.
+            - yarn publish --access=public --tag=next
```

Comment:
This is a temporary modification to the publish pipeline so that we will publish all releases with the `next` tag.

We can remove this once v3.0.0 is ready for a full release.